iac-extensions/docker/src/main/java/org/sonar/iac/docker/checks/utils/CommandDetector.java

```diff
@@ -253,6 +259,21 @@ public CommandDetector.Builder withPredicatesFrom(CommandDetector.Builder otherB
       return this;
     }
 
+    public CommandDetector.Builder withIncludeUnresolved(Predicate<String> predicate) {
+      addIncludeUnresolved(predicate, MATCH);
+      return this;
+    }
+
+    public CommandDetector.Builder withAnyExcludingIncludeUnresolved(Predicate<String> predicate) {
+      addCommandPredicate(new IncludingUnresolvedArgumentsPredicate(predicate, ZERO_OR_MORE));
+      return this;
+    }
+
+    public CommandDetector.Builder withArgumentResolutionIncludeUnresolved(Predicate<ArgumentResolution> predicate) {
```

Comment:
`withIncludingUnresolved` should be enough, as the parameter already describes the required generic

Reply:
`withArgumentResolutionIncludeUnresolved(Predicate<ArgumentResolution>)` -> `withIncludingUnresolved` and
`withIncludeUnresolved(Predicate<String>)` -> `withIncludingUnresolved`
is not possible because of weak Java typing :-|

Reply:
To bad :/